### PR TITLE
📖 book: add support dates

### PR DESCRIPTION
**Pull request by sbueringer**
Signed-off-by: Stefan Büringer buringerst@vmware.com

**What this PR does / why we need it**:
Just syncing the dates from main to release-1.1

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

### CODE REVIEW

**Review by killianmuldoon — APPROVED** [review]
/lgtm

**Comment by sbueringer** [comment]
Had to fix the old book too

**Comment by killianmuldoon** [comment]
I guess we should try to backport link updates to books for supported versions in future? Otherwise we'll always get these broken links when trying to make unrelated changes.

**Comment by sbueringer** [comment]
> I guess we should try to backport link updates to books for supported versions in future? Otherwise we'll always get these broken links when trying to make unrelated changes.

If these are fixes which also apply to previous versions - yes.

File: .gitignore [hunk 1]
```diff
@@ -61,6 +61,8 @@ vendor
 # User-supplied Tiltfile extensions, settings, and builds
 tilt.d
 tilt-settings.json
+tilt-settings.yaml
```
[the remark this answers is not in the document]
Comment (sbueringer) [reply .gitignore:64]: P.S. yes I rotated my AWS credentials :)

**Pull request merged**